### PR TITLE
Run tests against real redis on Travis

### PR DESCRIPTION
**Pull request by smn**

### CODE REVIEW

**Comment by smn** [comment]
Travis is not happy with this one https://travis-ci.org/praekelt/vumi-go/jobs/8500509.

**Comment by hodgestar** [comment]
Do we also get those errors if we run the test suite locally against real redis?

**Comment by smn** [comment]
Let me do this for Vumi itself first, https://github.com/praekelt/vumi/pull/578.

**Comment by hodgestar** [comment]
Where did this get to? Is it ready for review now that the Vumi side is done?

**Comment by smn** [comment]
I'm not sure where it got to :\ sorry. I'm guessing there's still more work to be done.

File: go/token/views.py [hunk 1]
```diff
@@ -1,24 +1,23 @@
...
 
 def token(request, token):
-    # This is special, since we don't necessarily have an authenticated user
-    # and we definitely don't have a conversation.
-    redis = RedisManager.from_config(settings.VUMI_API_CONFIG['redis_manager'])
-    tm = TokenManager(redis.sub_manager('token_manager'))
+    # We only need the redis manager here, but it's saner to get a whole
+    # vumi_api and not worry about all the setup magic.
+    api = vumi_api()
+    tm = TokenManager(api.redis.sub_manager('token_manager'))
```
Comment (hodgestar) [inline go/token/views.py]: Maybe the vumi_api should just get an token_manager attribute?

Comment (hodgestar) [inline go/token/views.py]: Certainly TokenManager.**init** doesn't look like it will take much time to run. :)

**Comment by hodgestar** [comment]
Other than the `vumi_api.token_manager` suggestion, :+1:.

**Comment by hodgestar** [comment]
:+1:

**Pull request merged**